src/index.ts

```diff
@@ -29,7 +29,7 @@ export interface SharedOptions {
   octokitAPIs?: OctokitAPIs;
 }
 
-export const DEFAULT_LABELS = 'autorelease: pending';
+export {DEFAULT_LABELS, RELEASE_PLEASE} from './constants';
```

Comment:
Given the odd behavior I encountered in `branch-name.ts` I wonder if we should even re-export these names here?